source/_patterns/04-pages/research-article/research-article~ar.yaml

```diff
@@ -95,6 +95,60 @@ body:
                     srcset: https://iiif.elifesciences.org/lax/38827%2Felife-38827-fig1-v2.tif/full/1234,/0/default.webp 2x, https://iiif.elifesciences.org/lax/38827%2Felife-38827-fig1-v2.tif/full/617,/0/default.webp 1x
                   - type: image/jpeg
                     srcset: https://iiif.elifesciences.org/lax/38827%2Felife-38827-fig1-v2.tif/full/1234,/0/default.jpg 2x, https://iiif.elifesciences.org/lax/38827%2Felife-38827-fig1-v2.tif/full/617,/0/default.jpg 1x
+            caption:
+              heading:
+                text: ألواح من الحجر الصخري تحمل Nanjinganthus.
+                level: 3
+              content:
+                - template: atoms-paragraph
+                  arguments:
+                    text:
+                      - طول كل القضبان 1 سم. (
+                      - template: atoms-bold
+                        arguments:
+                          attributes:
+                            dir: ltr
```

Comment:
Seem to be hitting https://www.w3.org/International/articles/inline-bidi-markup/#usecase6. Left in the bold as don't know the best solution.